README.md

```diff
@@ -16,29 +34,254 @@ Karpenter improves the efficiency and cost of running workloads on Kubernetes cl
 * **Removing** the nodes when they are no longer needed
 * **Consolidating** existing nodes onto cheaper nodes with higher utilization per node
 
-## Production Readiness Status
-- API-Version: AKS Karpenter Provider is currently in alpha (`v1alpha2`).
-- AKS-Feature State: [Node Auto Provisioning is currently in preview](https://learn.microsoft.com/en-gb/azure/aks/node-autoprovision?tabs=azure-cli)
 
-## Installation: Managed Karpenter (AKA Node Auto Provisioning) 
-The Node Auto Provisioning Preview, runs Karpenter as a managed addon similar to Managed Cluster Autoscaler. 
+## Node Auto Provisioning (NAP) vs. Self-hosted
+
+Karpenter provider for AKS can be used in two modes:
+* **[Node Auto Provisioning (NAP)](https://learn.microsoft.com/en-gb/azure/aks/node-autoprovision?tabs=azure-cli) mode** (preview): Karpenter is run by AKS as a managed addon similar to managed Cluster Autoscaler. This is the recommended mode for most users. Follow the instructions in Node Auto Provisioning documentation to use Karpenter in that mode.
```

Comment:
nit: I want `Node Auto Provisioning documentation` to have a link to the documentation, or at least the word `documentation` to.